src/components/offcanvas/OffcanvasHeader.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {omit} from 'ramda';
+import {default as RBOffcanvasHeader} from 'react-bootstrap/OffcanvasHeader';
+import {default as RBOffcanvasTitle} from 'react-bootstrap/OffcanvasTitle';
+
+/**
+ * Add a header to any offcanvas.
+ */
+const OffcanvasHeader = props => {
+  const {title, class_name, close_button, ...otherProps} = props;
+  return (
+    <RBOffcanvasHeader
+      className={class_name}
+      closeButton={close_button}
+      {...omit(['setProps'], otherProps)}
+    >
+      <RBOffcanvasTitle>{title}</RBOffcanvasTitle>
```

Comment:
Are there any circumstances under which you would want to add something to the header that isn't a title?

I guess it's also taking control of the close button etc.

Reply:
Header I believe should only contain the title and the close button. As per the other note, I have combined Header and Body into a single component, so these behaviours are now controlled from there.